app/forms/applicants/student_finance_form.rb

```diff
@@ -0,0 +1,16 @@
+module Applicants
+  class StudentFinanceForm < BaseForm
+    form_for Applicant
```

Comment:
Could these inherit from a BaseStudentFinanceForm to reduce duplication?
Or even share the same form?

Reply:
I did do that at first, and then reverted to to separate ones because it didn't feel like there was much of a benefit. I've made some changes since then though, so I'll have another go